js/web/lib/wasm/proxy-messages.ts

```diff
@@ -73,5 +73,10 @@ interface MesssageEndProfiling extends MessageError {
   in ?: number;
 }
 
+interface MessageIsOrtEnvInitialized extends MessageError {
```

Comment:
how/when is this used?

Reply:
It's used by the proxy-worker. Previously, InferenceSessionHandler file had a boolean that tracked whether OrtEnv is initialized. I wanted to make OrtEnvIsInitialized shared between training and inference, so I moved it to wasm-core-impl file, but all method calls from the InferenceSessionHandler to wasm-core-impl go through the proxy-wrapper, so I added support for calling the isOrtEnvInitialized method from a proxy.